docs/design/metrics.md

```diff
@@ -1,12 +1,59 @@
 # Metrics
 
-TODO:
+Operator provides metrics of its operations in a format supported by [Prometheus](https://prometheus.io/).
 
-- Investigate prometheus annotations wrt metrics
-    - see https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus-kubernetes.yml
-    - `prometheus.io/scrape`: Only scrape services that have a value of `true`
-    - `prometheus.io/scheme`: If the metrics endpoint is secured then you will need
-    - `prometheus.io/path`: If the metrics path is not `/metrics` override this.
-    - `prometheus.io/port`: If the metrics are exposed on a different port to the
+The metrics are exposed through HTTPS on port `8528` under path `/metrics`.
 
-- Add prometheus compatible `/metrics` endpoint to `arangod`
+For a full list of available metrics, see [here](./../generated/metrics/README.md).
+
+#### Contents
+- [Integration with standard Prometheus installation (no TLS)](#Integration-with-standard-Prometheus-installation-no-TLS)
+- [Exposing ArangoDB metrics](#ArangoDB-metrics)
+
+
+## Integration with standard Prometheus installation (no TLS)
+
+After creating operator deployment, you must configure Prometheus using a configuration file that instructs it
+about which targets to scrape.
+To do so, add a new scrape job to your prometheus.yaml config:
+```yaml
+scrape_configs:
+  - job_name:       'arangodb-operator'
+
+    scrape_interval: 10s # scrape every 10 seconds.
+
+    scheme: 'https'
```

Comment:
When no TLS then why the scheme is defined as HTTPS?

Default Prometheus installation does not support custom CAs, please check `spec.metrics.tls: false` for this.

Also, there are missing annotations used by prometheus.

Reply:
@ajanikow 
Current operator server runs only on HTTPS protocol.
If you request with HTTP proto you will get message `Client sent an HTTP request to an HTTPS server.`

`insecure_skip_verify: true` in Prometheus config allows to fetch data with no problems. Tested with latest stable `prom/prometheus` image

Reply:
`spec.metrics.tls` applies to ArangoDB sidecard-exporters, which can use HTTP

Reply:
Added info on TLS-enabled connection